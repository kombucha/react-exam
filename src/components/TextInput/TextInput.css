.TextInput {
  font-size: 1.5rem;
  font-family: inherit;
  background: none;
  border: none;
  outline: 0;
  border-bottom: 1px solid #feda4a;
  color: inherit;
  opacity: 0.9;
}

.TextInput:focus {
  opacity: 1;
}
