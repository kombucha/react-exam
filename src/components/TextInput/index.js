import React from "react";
import "./TextInput.css";

const TextInput = ({ className = "", ...props }) => (
  <input className={`TextInput ${className}`} {...props} />
);

export default TextInput;
