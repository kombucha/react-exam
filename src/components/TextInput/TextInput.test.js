import React from "react";
import TextInput from "./index";
import { shallow } from "enzyme";

it("should render properly", () => {
  const wrapper = shallow(<TextInput />);
  expect(wrapper).toMatchSnapshot();
});
