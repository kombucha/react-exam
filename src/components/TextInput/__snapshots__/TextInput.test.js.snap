// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TextInput />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "className": "TextInput ",
    },
    "ref": null,
    "rendered": null,
    "type": "input",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "className": "TextInput ",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
